unittests/runtime/Stdlib.cpp

```diff
@@ -155,3 +157,81 @@ SWIFT_CC(swift) SWIFT_RUNTIME_STDLIB_INTERNAL
 void *$Ss32_getErrorEmbeddedNSErrorIndirectyyXlSgSPyxGs0B0RzlF(void *) {
   abort();
 }
+
+// Hashable
+
+SWIFT_RUNTIME_STDLIB_INTERNAL
+const char $Ss8HashableMp[1] = {0};
+
+// Array
+
+SWIFT_RUNTIME_STDLIB_INTERNAL
+const char $SSaMn[1] = {0};
+
+// Dictionary
+
+SWIFT_RUNTIME_STDLIB_INTERNAL
+const char $Ss10DictionaryVMn[1] = {0};
+
+// Set
+
+SWIFT_RUNTIME_STDLIB_INTERNAL
+const char $Ss3SetVMn[1] = {0};
```

Comment:
If these are `char` they may not end up sufficiently aligned to meet the expectations of runtime or stdlib code that assume these structures are 4- or 8-byte aligned. Stubbing them out as `long long` would be safer.